.changeset/stupid-otters-listen.md

```diff
@@ -0,0 +1,17 @@
+---
+"@itwin/presentation-hierarchies-react": minor
+---
+
+Exposed `RootErrorRenderer`, which takes `renderRootError` property to render custom root error messages.
```

Comment:
Looks like this message needs to be updated

Reply:
changed